README.rst

```diff
@@ -45,15 +45,19 @@ The ``-w`` flag will actually implement the changes recommended by codespell. No
 
     codespell -I FILE, --ignore-words=FILE
 
-The ``-I`` flag can be used for a list of certain words to allow that are in the codespell dictionaries. The format of the file is one word per line. Invoke using: ``codespell -I path/to/file.txt`` to execute codespell referencing said list of allowed words. **Important note:** The list passed to ``-I`` is case-sensitive based on how it is listed in the codespell dictionaries. ::
+Get a list of words to ignore from ``FILE``. The ``FILE`` format is one word per line. **Important note:** Words are case-sensitive, based on how they are listed in the codespell dictionaries. ::
 
-    codespell -L word1,word2,word3,word4
+    codespell -L LIST, --ignore-words-list=LIST
 
-The ``-L`` flag can be used to allow certain words that are comma-separated placed immediately after it.  **Important note:** The list passed to ``-L`` is case-sensitive based on how it is listed in the codespell dictionaries. ::
+Ignore words from the comma-separated ``LIST``. **Important note:** Words are case-sensitive, based on how they are listed in the codespell dictionaries. Example:
 
-    codespell -S, --skip=
+* to ignore the words "fle" and "clos", invoke ``codespell --ignore-words-list="fle,clos"``.
```

Comment:
Can we make these examples match the config file examples below please?

Reply:
done